app/pages/project/talk-images.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import getSubjectLocation from '../../lib/get-subject-location.coffee';
+import TalkStatus from './talk-status';
+import Thumbnail from '../../components/thumbnail';
+
+export default class TalkImages extends React.Component {
+
+  render() {
+    return (
+      <div className="project-home-page__container">
+
+        {this.props.images.map((image) => {
+          const subject = getSubjectLocation(image);
+          return (
+            <div key={subject.id} className="project-home-page__talk-image">
+              <Thumbnail alt="" src={subject.src} format={subject.format} controls={false} />
```

Comment:
You should specify width and height for the Thumbnail here. Otherwise, it'll generate an image that's around 1000px x 1000px, which is way bigger than you need.